### PR TITLE
Py2 compatiblity

### PR DESCRIPTION
**Pull request by TCatshoek**
Added python 2 compatiblity fixes. Cloudomate should now be compatible with python 2 and 3.

### CODE REVIEW

**Comment by devos50** [comment]
@TCatshoek let me know when this PR is ready by mentioning me in a comment and prefixing the title of this pr with `READY`

File: cloudomate/hoster/vps/linevast.py [hunk 1]
```diff
@@ -79,6 +91,35 @@ def purchase(self, wallet, option):
         self._browser.submit_selected()
         self.pay(wallet, self.get_gateway(), self._browser.get_url())
 
+    def set_root_password(self, password):
```
Comment (Erackron) [inline cloudomate/hoster/vps/linevast.py]: This method was deliberately removed in favor of a different, unified root password change mechanism.'
Please do not readd it.

File: cloudomate/hoster/vps/clientarea.py [hunk 2]
```diff
@@ -86,19 +98,32 @@ def _login(self, email, password):
...
         :return: The clientarea homepage on succesful login.
         """
-        self._browser.open(self._url)
-        self._browser.select_form('.logincontainer form')
-        self._browser['username'] = email
-        self._browser['password'] = password
-        page = self._browser.submit_selected()
-        if "incorrect=true" in page.url:
-            print("Login failure")
+        service = self.get_specified_or_active_service()
```
Comment (Erackron) [inline cloudomate/hoster/vps/clientarea.py]: Is this change intentional or not?
If not, please revert

**Review by Erackron — CHANGES_REQUESTED** [review]

**Review by Erackron — APPROVED** [review]
